impress-org/givewp
# Feature: Embed Donation Form button

## Description

This PR adds a new feature to the Form Builder - Embed Button. This button allows folks to embed the form on a new or existing page. It also provides a pre-generated shortcode that the user can copy and embed by himself. 

## Affects
Form Builder

## Visuals

![Screen Capture_select-area_20231106121244](https://github.com/impress-org/givewp/assets/4222590/4b8732d8-b961-4751-80aa-bccc529430b0)

## Testing Instructions

1. Add a few new pages/posts
2. Go to the Edit form page and click on the new Embed button
3. Select page/post from the dropdown and insert Form into it
4. Copy the shortcode and paste it to some post/page
5. Verify that everything is working on the frontend and that form is inserted into posts/pages

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204808773418924
